x-pack/legacy/plugins/transform/public/app/lib/kibana/kibana_context.tsx

```diff
@@ -11,25 +11,21 @@ import {
   IndexPattern,
   IndexPatternsContract,
 } from '../../../../../../../../src/plugins/data/public';
-import { KibanaConfig } from '../../../../../../../../src/legacy/server/kbn_server';
-
-// set() method is missing in original d.ts
-interface KibanaConfigTypeFix extends KibanaConfig {
-  set(key: string, value: any): void;
-}
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { IUiSettingsClient } from '../../../../../../../../src/core/public/ui_settings';
```

Comment:
why is it restricted? 🤔 maybe it should go through shared imports?

Reply:
good point, I copied that over from somewhere. Looks like we can just use `import { IUiSettingsClient } from 'kibana/public';`. Fixed it in c5ae56a.